### PR TITLE
gif, footer and logo fixes

### PR DESCRIPTION
**Pull request by etsde**
- fixed the logo issues
- set max size for gif
- rounded gif frame
- new font styles
- footer with light font
- added commented: language toggle

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -21,51 +24,73 @@ main {
     background: none;
 }
 
-
 /* ################ Lesson component ################ */
 .lesson {
     background-color: #e9ecef;
 }
 
+/* ################ Footer ################ */
```
Comment (etsde) [inline css/style.css:32]: better-looking footer

File: css/style.css [hunk 2]
```diff
@@ -1,5 +1,8 @@
-
 /* ################ General stuff ################ */
```
Comment (etsde) [inline css/style.css:1]: I structured the css code a bit

File: css/style.css [hunk 3]
```diff
@@ -21,51 +24,73 @@ main {
...
-    src: url('/font/stolzl.bold.ttf') format('truetype'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+footer a, a:hover, a:focus, a:active {
+    text-decoration: none;
+    color: inherit;
 }
 
-@font-face {
-    font-family: 'stolzl-book';
-    src: url('/font/stolzl.book.ttf') format('truetype');
+/* ################ Cat Gif ################ */
```
Comment (etsde) [inline css/style.css:43]: put the cat gif in a rounded box

File: css/style.css [hunk 4]
```diff
@@ -21,51 +24,73 @@ main {
...
-    font-family: 'stolzl-light';
-    src: url('/font/stolzl.light.ttf') format('truetype');
+.rounded-box {
+    display: flex;
+    justify-content: center;
 }
 
-/* TODO: WOFF files - uncomment and add when available
+
+/* ################ Import Font ################ */
```
Comment (etsde) [inline css/style.css:56]: new fonts (also support for old browsers)

File: index.html [hunk 5]
```diff
@@ -48,25 +48,31 @@
         <div class="container-fluid">
             <div class="col-auto d-flex ">
                 
-                <a class="navbar-brand" href="#" onclick="window.location.href = window.location.href; return false;"><img src="img/logo.svg" width="45px" alt="" style="background: none;"> </a>
+                <a class="navbar-brand" href="#" onclick="window.location.href = window.location.href; return false;"><img src="img/smartplan-logo.svg" width="45px" alt="" style="background: none;"> </a>
```
Comment (etsde) [inline index.html:51]: logo fixes

(and Idk why I only can create review comments)

File: index.html [hunk 6]
```diff
@@ -48,25 +48,31 @@
...
-                <select class="form-select form-select select-timetable" aria-label="Dropdown class selector">
+            <div class="col-auto d-flex">
+                <select class="form-select select-timetable" aria-label="Dropdown class selector" style="width:fit-content;">
                     <option selected disabled>Select your class</option>
                     <optgroup label="IT 23/3">
                         <option value="it-23-3-A">C_IT 23/3 A</option>
                         <option value="it-23-3-B">C_IT 23/3 B</option>
                     </optgroup>
                 </select>
+
```
Comment (etsde) [inline index.html:64]: top secret features -,-

File: index.html [hunk 7]
```diff
@@ -98,17 +106,16 @@ <h2 class="weekday"> Welcome! </h2>
 
     <!-- Footer -->
     <footer class="text-center py-3 mt-auto">
-        <div class="container">
+        <div class="container footer">
```
Comment (etsde) [inline index.html:109]: better-looking footer

**Review by etsde — COMMENTED** [review]
please review

**Comment by etsde** [comment]
forgot changelog:


24/11/23
    - Shine bright like a diamond: Logo only from now
    - Because we love cats: cat gifs on start screen
    - Fixed some logo issues
    - Fixed some footer issues


can be added in next merge

File: .vscode/settings.json [hunk 8]
```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "main"
+    ]
+}
```
Comment (mr-woodapple) [inline .vscode/settings.json:5]: These do not belong into a repo. please remove and add to the .gitignore file.

File: css/style.css (hunk 4 above)
Comment (mr-woodapple) [reply css/style.css:56]: Are we using all of them?

**Pull request merged**